# SetNewTrainingExample: Updates the learner with a reward

# The state of the scene and command (un)validated
SceneState scene_state
Action action
bool good
---
# No return parameter

